packages/storage-plus/src/indexes.rs

```diff
@@ -6,17 +6,26 @@ use serde::{Deserialize, Serialize};
 
 use cosmwasm_std::{Binary, Order, StdError, StdResult, Storage, KV};
 
+use crate::keys::{EmptyPrefix, Prefixer};
 use crate::map::Map;
 use crate::prefix::range_with_prefix;
-use crate::{Bound, Endian};
+use crate::{Bound, Endian, Prefix, PrimaryKey};
+use std::marker::PhantomData;
 
 /// MARKER is stored in the multi-index as value, but we only look at the key (which is pk)
-const MARKER: u32 = 1;
+// FIXME: Re-introduce this for MultiIndex
+// const MARKER: bool = true;
```

Comment:
Where did this go?
btw, I used 1u32 as it has the smallest json representation (`1` vs `true`)

Reply:
I did this to make `prefix()` work. `prefix()` returns `Prefix<T>`, so I needed the value type to match.

That said, this is just temporary, and towards a solution.